components/molecules/charts/schedule/columnHelper.js

```diff
@@ -0,0 +1,199 @@
+import {
```

Comment:
@Ojaghi1991 Please remember the consistency - Doc Blocks - // React - style about UI and third party libraries above

Space code logicially etc

One export per file must be default

Move other large blocks to helper folder and seperate files also please